docs/sql-ref-syntax-ddl-create-table-datasource.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: global
+title: CREATE DATASOURCE TABLE
+displayTitle: CREATE DATASOURCE TABLE
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `CREATE TABLE` statement defines a new table using a Data Source. 
+
+### Syntax
+{% highlight sql %}
+CREATE TABLE [IF NOT EXISTS] [db_name.]table_name
+  [(col_name1 col_type1 [COMMENT col_comment1], ...)]
+  USING data_source
+  [OPTIONS (key1=val1, key2=val2, ...)]
+  [PARTITIONED BY (col_name1, col_name2, ...)]
+  [CLUSTERED BY (col_name3, col_name4, ...) [SORTED BY (col_name [ASC|DESC], ...)] INTO num_buckets BUCKETS]
+  [LOCATION path]
+  [COMMENT table_comment]
+  [TBLPROPERTIES (key1=val1, key2=val2, ...)]
+  [AS select_statement]
```

Comment:
I am trying to make all the docs follow the same convention: put a space in between the symbols (e.g. '|', '=', '[]') and text. Refer to sql-ref-syntax-ddl-create-database as an example.